README.md

```diff
@@ -23,9 +23,9 @@ To set up the sample application:
 2. Change into the `sfcc-sample-apps` folder:
 `cd sfcc-sample-apps`
 
-3. Copy the `api.example.js` file located at `/packages/storefront-lwc/scripts/`, save it as `api.js`, and make sure `api.js` is added to your `.gitignore` file.
+3. Copy the `api.example.mjs` file located at `/packages/storefront-lwc/scripts/`, save it as `api.mjs`, and make sure `api.mjs` is added to your `.gitignore` file.
```

Comment:
Currently the dev build is broken from a error swallowed by `lwc-services`. It relates to our native module usage with Node 12.16. `lwc-services` doesn't support esm modules very well. I'm going to try to fix that, but until then, it is easier to keep our native module support by switching these entry files to `.mjs`.